pkg/pod/entrypoint_lookup_impl_test.go

```diff
@@ -254,11 +256,55 @@ func TestBuildCommandMap(t *testing.T) {
 		wantErr: true,
 	}} {
 		t.Run(c.desc, func(t *testing.T) {
-			_, err := buildCommandMap(c.idx)
+			_, err := buildCommandMap(c.idx, true)
 			gotErr := (err != nil)
 			if gotErr != c.wantErr {
 				t.Fatalf("got err: %v, want err: %t", err, c.wantErr)
 			}
 		})
 	}
 }
+
+func mustConfig(t *testing.T, cfg v1.Config) v1.Image {
+	t.Helper()
+	img, err := mutate.Config(empty.Image, cfg)
+	if err != nil {
+		t.Fatal(err)
+	}
+	return img
+}
+
+func TestImageInfo(t *testing.T) {
+	for _, c := range []struct {
+		desc    string
+		img     v1.Image
+		hasArgs bool
+		wantCmd []string
+	}{{
+		desc:    "entrypoint-cmd-and-args",
+		hasArgs: true,
+		img: mustConfig(t, v1.Config{
+			Entrypoint: []string{"my", "entrypoint"},
+			Cmd:        []string{"my", "cmd"},
+		}),
+		wantCmd: []string{"my", "entrypoint"},
+	}, {
+		desc:    "entrypoint-cmd-and-no-args",
+		hasArgs: false,
+		img: mustConfig(t, v1.Config{
+			Entrypoint: []string{"my", "entrypoint"},
+			Cmd:        []string{"my", "cmd"},
+		}),
+		wantCmd: []string{"my", "entrypoint", "my", "cmd"},
+	}} {
+		t.Run(c.desc, func(t *testing.T) {
+			cmd, _, err := imageInfo(c.img, c.hasArgs)
+			if err != nil {
+				t.Fatalf("got err: %v", err)
+			}
+			if d := cmp.Diff(cmd, c.wantCmd); d != "" {
+				t.Errorf("Diff cmd %s", diff.PrintWantGot(d))
+			}
```

Comment:
```suggestion
			if d := cmp.Diff(c.wantCmd, cmd); d != "" {
				t.Errorf("Diff cmd %s", diff.PrintWantGot(d))
			}
```

Right? 🤔

Reply:
@imjasonh lol.. I always forget in which order to put them 😅